AUTHORS.md

```diff
@@ -4,12 +4,7 @@ This list is sorted by the number of commits per contributor in _descending_ ord
 
```

Comment:
This file is automatically modified so shouldn't be part of the PR.

Reply:
Should I remove from PR?

Reply:
Yes, please.